src/deluge/gui/l10n/english.json

```
@@ -454,6 +454,7 @@
         "STRING_FOR_POLY_FINGER_MPE": "Poly / finger / MPE",
         "STRING_FOR_REVERB_SIDECHAIN": "Sidechain",
         "STRING_FOR_ROOM_SIZE": "Room size",
+        "STRING_FOR_SUB": "Sub",
```

```suggestion
        "STRING_FOR_SUB_BANK_SHORT": "Sub",
```

for consistency with the other short horizontal menu strings